CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Version 2.5.0-beta2
-
+- Improved ServiceBus dependency and requests collection for (Microsoft.Azure.ServiceBus 2.1.0)
```

Comment:
Can you open a GH issue describing the enhancement (use content which is already given in the PR itself), and link to it here? 
The description provided is part of PR, and its not easy to find the PR which implement a given feature.

Reply:
Also, mention that 'Request' telemetry is not a typical request., and how to filter them out in UI. (using Source field..)

Reply:
Why are you saying that this `Request` is not real?

Reply:
@cijothomas I expect typical case would be that node with roleName 'worker-a' only reports requests from the service-bus, i.e. filtering them could be done by role name.
We don't really support filtering different requests types (as we don't have type property on the request).

I can describe how they can be filtered/grouped via Sourcefield. Where would you like me to write that?

Reply:
Ok, if processing of SB is done in worker roles. I was visualizing a scenario where this was done in WebRole itself, which has its own regular requests. So overview blades would show regular incoming 'Requests' and 'SB Requests'.
![image](https://user-images.githubusercontent.com/5232798/32752993-1886c758-c880-11e7-9b34-32548b3a6e2c.png)

Reply:
@SergeyKanzhelev The Request telemetry collected/created by this PR is not 'real' in the sense its not arising out of an Incoming User Request, and it does not have typical HTTP Response Codes. 
@lmolkova please correct me if this understanding is wrong.

Reply:
I can describe how they can be filtered/grouped via Sourcefield. Where would you like me to write that? -- in the GH issue to be opened describing this issue.
(The Changelog.md should link to the GH Issue , so its easier to find)

Reply:
@cijothomas for now we are not building UI to show SB requests, UI will remain the same as for HTTP requests. We might do this in future.
I think it's perfectly fine to consider them as a normal RequestsTelemetry (that represent server-side telemetry that describes operation processing). As you mentioned the only difference is that 'status code' makes no sense, but there are other instrumented protocols like that, e.g. ServiceFabric remoting

I'll update GH issue (#740) with details about source and reference it in the changelog

Reply:
@cijothomas - we've been manually collecting requests like that for workers (CDS, Iris) for at least 2 years now and the UX plays with it well even if some things are missing. This ServiceBus telemetry includes enough information to make the current UX light up nicely (i.e. name, timestamp, duration, success)

Reply:
@TomMilos yes i understand that. I was referring to users who might be doing SB read/process in a WebRoles where there are other web requests as well. They will suddenly start seeing additional requests (SB process) once upgraded to beta2, and this may be unexpected for many. Thats why i asked to have it explicitly called out in GH issue/changelogs.
(Esp. for asp.net core users, where this feature is just automatically enabled and somewhat invisible as well due to lack of AI.Config )

Reply:
@cijothomas I have described data structure in the #740 (and updated changelog to reference it), so customers have information on how to distinguish SB requests from the other. The assumption though is that in the majority of cases you have a web role and receive HTTP requests or you have a worker role and receive work items from the queue.

Reply:
Thanks!